ibis/bigquery/compiler.py

```diff
@@ -558,6 +558,24 @@ def log2(expr):
     return arg.log(2)
 
 
+@rewrites(ops.Sum)
+def bq_sum(expr):
+    arg = expr.op().args[0]
+    if isinstance(arg, ir.BooleanColumn):
+        return arg.cast('int64').sum()
```

Comment:
Here you actually need to pass `expr.op().args[1:]` to the `sum()` call because reductions take an additional where argument to allow filtering of values during aggregation.